recipe/bld.bat

```diff
@@ -1,51 +1,52 @@
 @echo on
-if "%ARCH%" == "64" (
-   set ARCH=x64
-) else (
-   set ARCH=Win32
-)
-
-:: The vc9 build uses a patch that added CMake support from a fork.
-:: We do not want to use it for vc14 since upstream provide a sln.
-
-:: Let us not bother with the bundled VS solution files since they
-:: do not build the CLI programs.
-:: if "%vc%" == "9" goto vc9_build
-:: goto vc14_build
-
-:vc9_build
-if "%vc%" NEQ "9" goto skip_c99_wrapper
-set COMPILER=-DCMAKE_C_COMPILER=c99-to-c89-cmake-nmake-wrap.bat
-set C99_TO_C89_WRAP_DEBUG_LEVEL=1
-set C99_TO_C89_WRAP_SAVE_TEMPS=1
-set C99_TO_C89_WRAP_NO_LINE_DIRECTIVES=1
-set C99_TO_C89_CONV_DEBUG_LEVEL=1
-COPY %LIBRARY_INC%\inttypes.h src\common\inttypes.h
-COPY %LIBRARY_INC%\stdint.h src\common\stdint.h
-:skip_c99_wrapper
+
+:: Build Static library (.lib)
+cmake -GNinja ^
+      -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+      %COMPILER% ^
+      -DCMAKE_C_USE_RESPONSE_FILE_FOR_OBJECTS:BOOL=FALSE ^
+      -DBUILD_SHARED_LIBS=OFF ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      --debug-trycompile ^
+      .
+ninja
+if errorlevel 1 exit /b 1
+
+ninja install
+if errorlevel 1 exit /b 1
+
+:: Rename Static library from liblzma.lib to liblzma_static.lib
+:: The next step will build the DLL, which creates an import library
+:: that would share the same name.
+move %LIBRARY_LIB%\liblzma.lib %LIBRARY_LIB%\liblzma_static.lib
+
+:: Build Shared library (.dll)
 cmake -GNinja ^
       -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
       %COMPILER% ^
       -DCMAKE_C_USE_RESPONSE_FILE_FOR_OBJECTS:BOOL=FALSE ^
+      -DBUILD_SHARED_LIBS=ON ^
       -DCMAKE_BUILD_TYPE=Release ^
       --debug-trycompile ^
       .
 ninja
 if errorlevel 1 exit /b 1
+
 ninja install
 if errorlevel 1 exit /b 1
+
 DEL src\common\inttypes.h
 DEL src\common\stdint.h
 goto common_exit
 
 :vc14_build
-cd Windows
+cd Windows\vs2017
```

Comment:
I think line 42 to 49 should be removed.
We don't need to build using the VS solution since we already have the artifacts built using cmake/ninja.

Reply:
Great. I went ahead and removed it. Thanks!